### ISSUE TITLE
buyCompleteSets fails with error code 0 (sometimes)

### ISSUE THREAD

**Issue by tinybike** [AugurProject/augur.js#129]
During test/reporting/sequence.js, `buyCompleteSets` sometimes fails with the following error:
```javascript
{"error":"0","message":"market not found"}
```

**Comment by tinybike** [comment]
This failure occurs after a market has been successfully created (and its market ID has been used to successfully look up its associated event ID):
```
Market ID: 0xedfdd17280e43efdb817ef21ac8efee47dbf6cec8044cfb1f141d815277f2c80
Event ID:  0x8f046c271b3bbcab25fd290517bdbc88bd46916d2eabcb678de249685ce0e964
```
Strangely, while the *event* is created without problems, the *market* is created, used to look up the event ID (implying that the market exists), then disappears:
```
> augur.getEventInfo("0x8f046c271b3bbcab25fd290517bdbc88bd46916d2eabcb678de249685ce0e964")
[ '0xde30f278936b0c4bd4d23122b2de95e2d1750cd361febfadb9a34dd434dd732f',
  '1469257201',
  '0',
  '1',
  '2',
  2,
  '4.5' ]
> augur.getMarketInfo("0xedfdd17280e43efdb817ef21ac8efee47dbf6cec8044cfb1f141d815277f2c80")
null
```
Even the `getMarketEvent` request that succeeds initially (fetching the event ID) subsequently fails for the market:
```
> augur.getMarketEvent("0xedfdd17280e43efdb817ef21ac8efee47dbf6cec8044cfb1f141d815277f2c80", 0)
'0x0000000000000000000000000000000000000000000000000000000000000000'
```

**Comment by tinybike** [comment]
Fixed in AugurProject/augur-core@7615733565d1a540a5d53b0a02e21c1a38a49b43
